includes/class-newspack-popups.php

```diff
@@ -388,5 +389,28 @@ public static function popup_default_fields( $post_id, $post, $update ) {
 		update_post_meta( $post_id, 'trigger_scroll_progress', 30 );
 		update_post_meta( $post_id, 'utm_suppression', '' );
 	}
+
+	public static function create_lightweight_api_config() {
+		$config_path = WP_CONTENT_DIR . '/../newspack-popups-config.php';
+		$has_config  = file_exists( $config_path );
+		if ( false === $has_config ) {
```

Comment:
Here sharing a general and simple coding style personal preference -- some fun stuff -- but not asking you to change anything here if you don't want to.

I generally always prefer using the early return pattern, instead of nesting code blocks into `if`s, whenever possible (in all coding languages). I'm sure this is just a personal preference, and therefore not very important, but for example. instead of:
```
function() {
  if ( something ) {
     do stuff
  }
}
```
I always try and use this:
```
function() {
  if ( ! something) {
    return
  }

  do stuff
}
```

This particular style:
- removes a whole nesting block and, makes code a bit more readable -- especially if there's another block/conditional/loop in the `do stuff` block
- it structures functions to immediately express when an exit/return is needed, which makes that flow condition a bit more readable, instead of having to read through the whole function -- this here is just a general observation, not related to your example at all

About this "pattern":
- [link 1](https://medium.com/swlh/return-early-pattern-3d18a41bba8#:~:text=Return%20early%20is%20the%20way,when%20conditions%20are%20not%20met.)
- [link 2](https://forum.freecodecamp.org/t/the-return-early-pattern-explained-with-javascript-examples/19364)
- [link 3](https://medium.com/better-programming/are-early-returns-any-good-eed4b4d03866)